web/concrete/single_pages/dashboard/reports/logs.php

```diff
@@ -67,12 +67,18 @@
                     ?></td>
                     <td valign="top"><strong><?=$ent->getType()?></strong></td>
                     <td valign="top"><strong><?php
-                    if($ent->getUserID() == NULL){
+                    $uID = $ent->getUserID();
+                    if(empty($uID)) {
                         echo t("Guest");
                     }
-                    else{
-                        $u = User::getByUserID($ent->getUserID());
-                        echo $u->getUserName();
+                    else {
+                        $u = User::getByUserID($uID);
+                        if(is_object($u)) {
+                            echo $u->getUserName();
+                        }
+                        else {
+                            echo tc('Deleted user', 'Deleted (id: %s)', $uID);
```

Comment:
IMHO that's fine to have a context, who cares? Replacing it with a comment may cause problems.
I mean, if we have for instance these two strings in two different places:
```php
echo t(/*i18n: Deleted page*/'Deleted (id: %s)', $cID);
echo t(/*i18n: Deleted user*/'Deleted (id: %s)', $uID);
```
Because the two translated strings are the same, we can have just one translation.
But (at least in Italian) we'd need to have two different translations...